tests/components/recorder/conftest.py

```diff
@@ -46,3 +50,70 @@ def recorder_dialect_name(hass: HomeAssistant, db_engine: str) -> Generator[None
             "homeassistant.components.recorder.Recorder.dialect_name", db_engine
         ):
             yield
+
+
+@dataclass
+class InstrumentedMigration(slots=True):
```

Comment:
```suggestion
@dataclass(slots=True)
class InstrumentedMigration:
```

misfire on the line on previous suggest